tests/robot-tests/tests/general_public/table_tool_exclusions_by_geographic_level.robot

```diff
@@ -83,7 +83,7 @@ Validate the query could exceed the maximum allowable table size
     user waits until page contains
     ...    Could not create table as the filters chosen may exceed the maximum allowed table size.
     user waits until page contains    Select different filters or download the subject data.
-    user waits until page contains button    Download Exclusions by geographic level (csv, 512 B)    %{WAIT_MEDIUM}
+    user waits until page contains button    Download Exclusions by geographic level (csv, 37 MB)    %{WAIT_MEDIUM}
```

Comment:
We'll just need to issue an update to the local developer db volume to change the file entry to match this same filesize.